pkg/controller/restore_controller.go

```diff
@@ -85,6 +83,11 @@ type restoreController struct {
 	newBackupStore   func(*api.BackupStorageLocation, persistence.ObjectStoreGetter, logrus.FieldLogger) (persistence.BackupStore, error)
 }
 
+type apiRestoreResult struct {
+	warnings api.RestoreResult
+	errors   api.RestoreResult
```

Comment:
Please move the `errors` var to the same line as `warnings`.

Reply:
which means to declear single field of apiRestoreResult to be mixing warnings with errors? or to be declaring as an array?

Reply:
Yes, the first, just move the variable so both are on the same line, like so: `warnings, errors api.RestoreResult`